internal/pkg/infrastructure/postgres/sql.go

```diff
@@ -496,6 +493,32 @@ func constructWhereCondWithTimeRange(lowerLimitTimeRangeCol, upperLimitTimeRange
 	return strings.Join(conditions, " AND ")
 }
 
+// constructWhereNamedArgCondWithTimeRange constructs the WHERE condition for the given columns with time range
+// if arrayColNames is not empty, ANY operator will be added to accept the array argument for the specified array col names
```

Comment:
Add a sample here in the comment for better understanding how the given parameters may construct the where condition.